### ISSUE TITLE
AGN modeling for DDF

### ISSUE THREAD

**Issue by katrinheitmann** [LSSTDESC/DC2-production#377]
On the desc-dc2-agn slack channel, a new discussion was started with regard to the AGN modeling for the DDFs. This problem needs to be addressed before we can start generating instance catalogs and then the simulations. Thanks to @jiwoncpark for the post that I am copying below: 

From Ji Won: I'm working on validating the AGN variability model, and am finding that the DC2 AGN are ~2.5 mag less luminous than the SDSS S82 AGN because the Eddington ratio from cosmoDC2 is lower. I've laid out some investigative plots here https://docs.google.com/presentation/d/1Ozh7yONbOoNA1Pun-5ISpGRYoluOoXHrDFpYScimtJw/edit?usp=sharing and the relevant repo is https://github.com/jiwoncpark/damped-random-walk. @franzebauer It would be great if you could help me understand the difference in Eddington ratio distributions. Thank you!

From Franz: Hi. To make a proper comparison, I think you should compare identical quantities (e.g., sources / deg-2) on the same scale.
For the histogram, for example, it would be useful to plot things in log space so we can see how different the bright end really is. Is DC2 delivering the same number density of bright QSOs as SDSS stripe82? Most critically, the DC2 variability catalog extends ~3-4 mags deeper (i.e., ~30x) than the SDSS stripe82 one, so I am not initially that surprised that the peak of the histograms are different by 2.5 mags.
Regarding the other plots, again, I think the point of comparison needs to be is DC2 delivering the appropriate number density of sources in a given region of phase space? The MacLoed sample is going to have selection effects so that it only includes the brightest (highest L/Ledd) AGN, while DC2 should push down to weaker intrinsic accretion. So having the peaks in various phase spaces be different is perhaps to be expected.

I hope we can make some progress on this soon and tracking it on github is probably easier than on slack. Thanks all!

**Comment by franzebauer** [comment]
(for me, tracking on slack is easier, but I will try to pay attention here...). At the moment, I was waiting for a response from Ji Won to report on relative number densities at the bright end.

**Comment by katrinheitmann** [comment]
@jiwoncpark and @franzebauer: Has this been all now resolved (I think there are some slides from one of the telecons showing some tests). If so, could you put a link to the final tests and close the issue? Thanks!

**Comment by franzebauer** [comment]
I think it is addressed or understood now, yes.

**Comment by jiwoncpark** [comment]
@katrinheitmann The [same slide deck](https://docs.google.com/presentation/d/1Ozh7yONbOoNA1Pun-5ISpGRYoluOoXHrDFpYScimtJw/edit?usp=sharing) has been updated with the final set of plots. The central values of tau (which was the biggest offender) now agree. The most important are the conclusion slides with the cornerplots (p.17 - 24).

I believe [my PR](https://github.com/LSSTDESC/sims_GCRCatSimInterface/pull/80) on `sims_GCRCatSimInferface` that makes the necessary changes in the AGN model still needs merging.

**Comment by jchiang87** [comment]
@jiwoncpark Are all of the changes you need to make to get tau to agree are in that PR?  When I asked on Dec 5, you were still trying to get things to work.  The most recent commit on that PR is dated Nov 20.

**Comment by jiwoncpark** [comment]
@jchiang87 Yes! I've double-checked the final commit. Sorry for the delay; the tau problem seems addressed and I wasn't sure whether the 0.25 dex discrepancy in SF_inf could be ignored. I was working on interpreting the results -- not changing the model further.

**Comment by katrinheitmann** [comment]
Since we finalized the AGN modeling for Run 3.0i and those simulations have started now, I close this issue. If we find any issues with the AGNs (and I very much hope not), we should start a new issue. If you feel strongly about not closing this issue, please reopen and state further plans. Thanks!
